src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Icon } from "@chakra-ui/react"
+import { RiInformationLine } from "react-icons/ri"
+import TooltipComponent from "."
+
+type TooltipType = typeof TooltipComponent
+
+const meta = {
+  title: "Molecules / Overlay Content / Tooltip",
+  component: TooltipComponent,
+} satisfies Meta<TooltipType>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Tooltip: Story = {
+  args: {
+    content: "text here",
+    children: <Icon as={RiInformationLine} />,
+  },
+}
```

Comment:
Could you add a new story that shows how to use rich content (e.g. a text with links)?

Reply:
Done!